homeassistant/components/pi_hole/strings.json

```diff
@@ -25,5 +25,11 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
     }
+  },
+  "issues": {
+    "deprecated_yaml": {
+      "title": "The PI-Hole YAML configuration is being removed",
```

Comment:
Note that this means that we should include translation updates when we make the final release.

Reply:
Oh I guess german and English translations are included, those will be cherry-picked along.